### PR TITLE
remove inline css

### PR DESCRIPTION
**Pull request by xi**
This removes the last traces of inline style so the CSP can restricted and the application gets a bit more secure. See also #241

### CODE REVIEW

File: static/css/main.css [hunk 1]
```diff
@@ -122,3 +122,31 @@ table tr th, table tr td {
...
+meter.progress::-webkit-meter-bar {
+    border: 0;
+    border-radius: 0;
+}
+meter.progress::-webkit-meter-optimum-value {
+    background: var(--primary);
+}
+meter.progress::-moz-meter-bar {
+    background: var(--primary);
+}
```
Comment (xi) [inline static/css/main.css:152]: I tested this in firefox and chrome, but still this is all hacky and brittle. IMHO we could also use `<meter>` without bootstrap styling.

Comment (phillipthelen) [reply static/css/main.css:152]: How hacky and brittle is it? Like in what way does it break? I think if it mostly works and "breaks" by falling back to a more default styling, that would probably be okay?

Comment (xi) [reply static/css/main.css:152]: I think it is unlikely that this breaks in a way that affects other elements on the page, but it could well be that the relevant part of the bar is not colored.

I would feel safer with reverting d2060da325e0a20583261a38fb4b551cd88b671d. However, other people tent to value esthetics higher than I do.

Comment (phillipthelen) [reply static/css/main.css:152]: I'll try to also test it in safari and maybe edge. If it also works there, I'd just go ahead with it and deploy it. the bar isn't used a lot and also has numbers on it, so even if it breaks it would be noticeable and we could still make changes.
